src/components/MyKiva/MyKivaSharingModal.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+	<div>
+		<KvLightbox
+			:visible="isVisible"
+			title="Invite a friend to Kiva!"
+			@lightbox-closed="closeLightbox"
+		>
+			<div class="tw-relative">
+				<textarea
+					class="tw-w-full tw-border tw-border-tertiary tw-rounded-sm tw-h-12 tw-p-2"
+					style="height: 10rem;"
+					v-model="shareMessage"
+				>
+				</textarea>
+				<KvMaterialIcon
+					class="tw-w-2.5 tw-h-2.5 tw-absolute tw-bottom-2 tw-right-2"
+					:icon="mdiPencilOutline"
+				/>
+			</div>
+			<div
+				class="tw-flex tw-mt-2.5 tw-flex-wrap"
+			>
+				<KvButton
+					variant="ghost"
+					class="social-button"
+					v-kv-track-event="['portfolio', 'share', 'facebook', utmCampaign]"
+					@click="showSharePopUp(
+						facebookShareUrl({utmCampaign, utmContent}),
+						'Thanks for sharing to Facebook!')"
+				>
+					<KvMaterialIcon
+						class="social-button__icon social-button__icon--facebook"
+						:icon="mdiFacebook"
+					/>
+					<span class="tw-font-medium">Facebook</span>
+				</KvButton>
+				<KvButton
```

Comment:
Could you add a new ticket to the same epic and current sprint for replacing this with bluesky?

Reply:
After merging to have an url reference...